### ISSUE TITLE
Have CLI and UI option to export extra PDF's for drilldown into grouped resources

### ISSUE THREAD

**Issue by Shocktrooper** [Pluralith/pluralith-cli#79]
I noticed that the graph to PDF only supports the current view even if there are grouped resources in the graph.  It would be useful if there was something like a --export-all option that exported another PDF for each subview.  The exported PDF can then have markdown style relative hyperlinks that open up the other pdf's when clicked on.

I have tested that this works by manually editing a PDF and it would be really cool to see this happen

**Comment by DanThePutzer** [comment]
@Shocktrooper didn't know one could link PDFs on the same path like that!

Great suggestion, could make things a lot more interactive! We'll look into it and report back!

**Comment by Shocktrooper** [comment]
Also one thing to note is that it appears sub group diagrams cannot be currently exported at all in the desktop

**Comment by DanThePutzer** [comment]
@Shocktrooper yes that's correct, we haven't gotten around to implementing that.
